pages/api/auth/[...nextauth].ts

```diff
@@ -103,6 +103,7 @@ export const authOptions = {
     async jwt({ token, user }: { token: JWT; user?: QueriedAdminData | any }) {
       if (user) {
         token.uid = user._id;
+        token.name = user.firstName + ' ' + user.lastName;
         token.status = user.status;
       }
       return token;
```

Comment:
Use session to store user data. Can be handy whenever you need frequent access to user data.